Broken Links Detected on 2025-12-02

### 🚨 Broken Links Detected in Moonbeam Docs

The scheduled link checker has found one or more broken links.
This issue was automatically generated by the [central docs workflow](https://github.com/papermoonio/workflows/actions/runs/19858141418).

---

#### 📝 Link Checker Report:

# Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 76711 |
| ✅ Successful | 75881 |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 827   |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 3     |

## Errors per input

### Errors in moonbeam-mkdocs/site/builders/integrations/indexers/covalent/index.html

* [ERROR] [https://moonbase-minterc20.netlify.app/](https://moonbase-minterc20.netlify.app/) | Error (cached)

### Errors in moonbeam-mkdocs/site/builders/get-started/networks/moonbase/index.html

* [ERROR] [https://moonbase-minterc20.netlify.app/](https://moonbase-minterc20.netlify.app/) | Network error: error sending request for url (https://moonbase-minterc20.netlify.app/) Maybe a certificate error?

### Errors in moonbeam-mkdocs/site/tutorials/interoperability/remote-batched-evm-calls/index.html

* [ERROR] [https://moonbase-minterc20.netlify.app/](https://moonbase-minterc20.netlify.app/) | Error (cached)
[Full Github Actions output](https://github.com/papermoonio/workflows/actions/runs/19858141418?check_suite_focus=true)

---
#### Next Steps
1.  **Investigate**: Check if the links are temporarily down or permanently gone.
2.  **Fix**: Update the source markdown files in this repository.
3.  **Close**: Once fixed, this issue can be closed.

--- a/.urlignore
+++ b/.urlignore
@@ -17,6 +17,7 @@ https://x.com/
 
 # Ignore known working URLs
 https://ethereum.org/
+https://moonbase-minterc20.netlify.app/
 
 # 405 Method not Allowed
 https://apps.moonbeam.network/

